### PR TITLE
143 - Add message table component and message value pipe

### LINKED ISSUE #143: Remove "raw" from "Raw message" card and make a paginated list (table) of messages with custom ranges

### PR DESCRIPTION
**Pull request by darkodraskovic**
Resolves #143 partially.

Signed-off-by: Darko Draskovic <darko.draskovic@gmail.com>

### CODE REVIEW

File: src/app/shared/message-table/value.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { MainfluxMsg } from 'app/common/interfaces/mainflux.interface';
+
+@Pipe({name: 'valuePipe'})
```
Comment (manuio) [inline src/app/shared/message-table/value.pipe.ts:5]: Simply `value`?

Comment (darkodraskovic) [reply src/app/shared/message-table/value.pipe.ts:5]: It seemed to me that name would be prone to conflict, but since it's pipe, probably it's ok.

**Review by manuio — APPROVED** [review]
LGTM

File: src/app/shared/message-table/value.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { MainfluxMsg } from 'app/common/interfaces/mainflux.interface';
+
+@Pipe({name: 'value'})
```
Comment (manuio) [inline src/app/shared/message-table/value.pipe.ts:5]: Good  idea this piepe, but after to  think about this should probably go in `commons` dir. `shared` is for shared components. 
You could also rename this `msgValue` to be less ambiguous.

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]

**Pull request merged**